web/src/main/java/no/nav/modiapersonoversikt/rest/person/PersonsokController.kt

```diff
@@ -376,9 +376,9 @@ fun PersonsokRequest.tilPdlKriterier(clock: Clock = Clock.systemDefaultZone()):
     }
 
     return listOf(
-        PdlKriterie(PdlFelt.NAVN, navn),
+        PdlKriterie(PdlFelt.NAVN, navn, searchHistorical = true),
```

Comment:
Om vi introduserer `PdlSokeOmfang` som en enum-class her så tenker jeg vi kan gjøre:

```suggestion
        PdlKriterie(PdlFelt.NAVN, navn, PdlSokeOmfang.HISTORISK_OG_GJELDENDE),
```